### DISCUSSION TITLE
After restart

### DISCUSSION THREAD

**Discussion by rwcloudridgeworks** [amidaware/tacticalrmm#1258] [Q&A]
after a restart, i get the below errors 

npm ERR! code ENOLOCK
npm ERR! audit This command requires an existing lockfile.
npm ERR! audit Try creating one first with: npm i --package-lock-only
npm ERR! audit Original error: loadVirtual requires existing shrinkwrap file

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-08-23T11_09_27_491Z-debug-0.log

**Comment by dinger1986** [comment]
a few users seem to be having this issue after the latest update, restore your most recent backup.

**Comment by wh1te909** [comment]
need more info please follow the template. what OS are you on for starters?
